Source/bmalloc/Configurations/bmalloc.xcconfig

```diff
@@ -21,6 +21,13 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
 
+// Do not export a modulemap unless specifically configured to do so
+// using MODULARIZE_WTF_AND_BMALLOC=YES. A modulemap significantly
+// affects how downstream components build, so we don't yet enable it
+// by default.
+EXCLUDED_SOURCE_FILE_NAMES_ = module.modulemap;
+EXCLUDED_SOURCE_FILE_NAMES = $(inherited) $(EXCLUDED_SOURCE_FILE_NAMES_$(MODULARIZE_WTF_AND_BMALLOC));
```

Comment:
Per our discussion, I think we'll want to remove the `MODULARIZE_WTF_AND_BMALLOC` setting, make this behavior unconditional, and wait to land on main until it's safe to do so. Especially since emitting the modulemap does not properly invalidate incremental builds.